newsroom/public/views.py

```diff
@@ -2,6 +2,12 @@
 from newsroom.public import blueprint
 
 
-@blueprint.route('/copyright')
-def copyright():
-    return flask.render_template('copyright.html')
+@blueprint.route('/privacy')
+def privacy():
+    return flask.render_template('privacy.html')
+@blueprint.route('/terms')
```

Comment:
I think you need 2 blank spaces before @blueprint otherwise flake8 will faill

Reply:
Thanks I will try adding 2 blank spaces before @blueprint.